### PR TITLE
Feature/fill shops 2

### PR DESCRIPTION
**Pull request by Ishadijcks**
Add items to all the shops.

Currency images will be added on another branch

### CODE REVIEW

File: src/scripts/items/BattleItem.ts [hunk 1]
```diff
@@ -4,9 +4,31 @@ class BattleItem extends Item {
...
+                break;
+            case GameConstants.BattleItemType.Token_collector:
+                basePrice = GameConstants.ItemPrice.Token_collector;
+                break;
+            case GameConstants.BattleItemType.Item_magnet:
+                basePrice = GameConstants.ItemPrice.Item_magnet;
+                break;
+            case GameConstants.BattleItemType.Lucky_incense:
+                basePrice = GameConstants.ItemPrice.Lucky_incense;
+                break;
```
Comment (Aegyo) [inline src/scripts/items/BattleItem.ts]: Could this be simplified to `GameConstants.ItemPrice[ GameConstants.BattleItemType[type] ]` for every case?
Similarly for the other item types.

File: src/scripts/shop/Shop.ts [hunk 2]
```diff
@@ -4,8 +4,14 @@
 class Shop {
     items: KnockoutObservableArray<Item>;
```
Comment (Aegyo) [inline src/scripts/shop/Shop.ts]: Why an observable array? As far as I know, the items in a shop will always be the same, so I don't see what benefit we get from making the array observable.

Comment (Ishadijcks) [inline src/scripts/shop/Shop.ts]: Ah I think it already was, but you're right

File: src/scripts/items/BattleItem.ts (hunk 1 above)
Comment (Ishadijcks) [inline src/scripts/items/BattleItem.ts]: I remember trying that and it not working, but I'll look into it again

**Comment by Ishadijcks** [comment]
Run `Save.initializeItems` and `Save.initializeMultipliers()`

**Comment by Aegyo** [comment]
Ah, that fixed the numbers. Battle item images are named like Xattack, but the item names in code are like xAttack

**Comment by Ishadijcks** [comment]
Stupid windows and its case insensitiveness

**Pull request merged**